src/main/java/bio/terra/service/profile/ProfileRequestValidator.java

```diff
@@ -29,6 +30,10 @@ public void validate(@NotNull Object target, Errors errors) {
                 errors.rejectValue("billingAccountId",
                     "The id must be 3 sets of 6 capitalized alphanumeric characters separated by dashes");
             }
+            if (StringUtils.isEmpty(billingProfileRequestModel.getId())) {
```

Comment:
Is this not something that can be automated by the code generation?

Reply:
Yes, once MY gets his PR in. I'll take it out.

Reply:
Looks like mine is going in first so i'll leave it in for now. We need a pass over all of the validations to remove ones that are unneeded.